src/papers/smart-token-paper.md

```diff
@@ -97,13 +97,15 @@ In essence, the trajectory of Web 2.0 innovation is not solely constrained by th
 
 ## Reimagining Web 3.0: Beyond the Internet of Value
 
-The prevailing vision for Web 3.0 is characterised as an 'Internet of Value' - a platform where value, in its various forms, is distributed more equitably among users and creators, breaking away from the monopolistic tendencies of the Web 2.0 era. This centralisation of value, in this view, is seen as the root cause of many of the issues plaguing the current web ecosystem, from stifled innovation to privacy concerns.
+The dominant vision of Web 3.0 heralds it as an 'Internet of Value'—a digital ecosystem where decentralized financial systems ensure equitable value distribution, challenging the monopolistic centralisation of Web 2.0. This concentration of value is often cited as the principal catalyst for the myriad challenges afflicting the current web ecosystem.
```

Comment:
The argument that the centralization of value is a consequence rather than a cause is well-articulated. However, it would benefit from providing specific examples or evidence to support this claim. E.g add a quote/reference for proving the argument.

Reply:
Hi don't use ai without providing the full paper;)